packages/storybook/src/stories/workspace-avatar.stories.tsx

```diff
@@ -54,6 +54,9 @@ fetch(new URL('@affine-test/fixtures/smile.png', import.meta.url))
       new Blob([buffer], { type: 'image/png' })
     );
     avatarBlockSuiteWorkspace.meta.setAvatar(id);
+  })
+  .catch(() => {
+    // just ignore
```

Comment:
I believe that unknown errors should not be ignored, even in the storybook environment.

```suggestion
```